content/content-strategy/manage-content-requests/create-content/pair-writing/how.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: text/textblock
+section: How to do pair writing
+---
+### [1]Step 1: Start with user needs
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3.

Reply:
Trisha: done